lib/Client.js

```diff
@@ -1579,6 +1678,34 @@ class Client extends EventEmitter {
         return this.requestHandler.request("POST", Endpoints.CHANNEL_FOLLOW(channelID), true, {webhook_channel_id: webhookChannelID});
     }
 
+    /**
+    * Get all active threads in a channel
+    * @arg {String} channelID The ID of the channel
+    * @returns {Promise<Array<NewsThreadChannel | PrivateThreadChannel | PublicThreadChannel>>}
+    */
+    getActiveThreads(channelID) {
+        return this.requestHandler.request("GET", Endpoints.THREADS_ACTIVE(channelID), true).then((channels) => channels.map((c) => Channel.from(c, this)));
```

Comment:
```suggestion
        return this.requestHandler.request("GET", Endpoints.THREADS_ACTIVE(channelID), true).then((response) => response.threadsmap((c) => Channel.from(c, this)));
```
the request returns an object, like for `getArchivedThreads`

Reply:
This isn't the correct suggestion. We don't want to just return the channels. I'll push a commit soon